Marlin/pins.h

```diff
@@ -1309,6 +1315,7 @@
        #endif
      #else
        #define BTN_ENC 16  //the click switch
+       #define LCD_SDSS 28 //to use the SD card reader on the smart controller rather than the melzi board
```

Comment:
All gfx panels have an additional SD Card slot. Maybe could be useful to foresee a define to enable/disable the internal SD slot. This is the meaning of this pin: use the external sd slot.